### DISCUSSION TITLE
how to load spine files?

### DISCUSSION THREAD

**Discussion by tonku321** [pixijs/pixijs#9526] [v8 General]
Using pixi.js version 7.2.4 we can load spine asset like this `PIXI.Assets.load("spine/spineboy-pro.json")`
and response is `{ spineData: SkeletonData3, spineAtlas: TextureAtlas }` which later can be passed to "pixi-spine@4.0.4" package to create working animation.

But in pixi.js version 8.0.0-alpha.1 method `PIXI.Assets.load("spine/spineboy-pro.json")` just parses the json file and returns it as object `{skeleton: {…}, bones: Array(67), slots: Array(52), ik: Array(7), transform: Array(8), …}`.

what changes in `PIXI.Assets.load` method led to this behavior?
how can i reproduce the same response in the new version?

**Comment by bigtimebuddy** [comment]
I don't think Spine is supported yet in v8. Is that right @GoodBoyDigital?

**Comment by zeeeeby** [comment]
@bigtimebuddy @GoodBoyDigital hey? is there any update? i have faced with this issue, and don't know what to do. I am using package @pixi-spine/all-4.1

**Reply by tonku321** [reply]
"@esotericsoftware/spine-pixi-v8": "^4.2.93" works with "pixi.js": "^8.13.2"

**Reply by zeeeeby** [reply]
> @pixi-spine/all-4.1

does @esotericsoftware/spine-pixi-v8 work with spine 3.8 or 4.1?

**Reply by tonku321** [reply]
I don't know, but I'm using Spine 4.2.37 with setup that I described above
